addons/csi/hetzner/hcloud-csi-1.6.0.yaml

```diff
@@ -364,6 +364,22 @@ spec:
     - port: 9189
       name: metrics
       targetPort: metrics
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: hcloud-csi
+  namespace: kube-system
+spec:
+  podSelector:
+    matchLabels:
+      app: hcloud-csi
+  policyTypes:
+    - Egress
+  egress:
```

Comment:
Isn't this same as `egress: {}`?

Reply:
same question :) If they are same, I would prefer to use the same expression everywhere, ideally the one which will be compatible with IPv6 ;)

Reply:
No, this does not allow egress traffic to the cluster. But let me double check if I am wrong I will let you know

Reply:
> No, this does not allow egress traffic to the cluster

I don't follow. Which cluster are we talking about?

Reply:
So do we already know what is the difference between:
```
egress:
    - {}
```
and 
```
  egress:
    - to:
        - ipBlock:
            cidr: 0.0.0.0/0
```
?

Reply:
It is the same, the networkpolicy editor is a bit confusing about it. What may makes sense here is to use cidr: 0.0.0.0/0 and use `except` and deny the cluster cidr. That way all egress to world would be allowed, but cluster access denied.